MetricsUploader/include/MetricsUploader/MetricsUploader.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+#define ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+
+#ifdef _WIN32
+#include <windows.h>
+#endif  // WIN32
+
+#include "orbit_log_event.pb.h"
+
+namespace orbit_metrics_uploader {
+
+/**
+ * This class is used for sending log events from Orbit. It works only on Windows and only if
+ * metrics_uploader_client.dll is available. The types of logs that could be send are defined in
+ * orbit_client_protos::OrbitLogEvent::LogEventType.
+ */
+class MetricsUploader {
+ public:
+  // Load metrics_uploader_client.dll if available and starts metrics uploader client.
+  MetricsUploader();
+
+  // Unload metrics_uploader_client.dll
+  ~MetricsUploader();
+
+  // Send log events to the server using metrics_uploader.
+  void SendLogEvent(orbit_client_protos::OrbitLogEvent_LogEventType log_event_type);
+
+ private:
+  inline static bool is_init_{false};
+#ifdef _WIN32
+  HINSTANCE metrics_uploader_client_dll_;
```

Comment:
Nit: AFAIK the return type of LoadLibrary is HMODULE (which happens to be the same as HINSTANCE - but ideally we don't assume that.)

Reply:
Right, fixed it, than you!